classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
```

Comment:
- [ ] It would be nice if you test this method because you are supposed to test all the methods in the classes so it would be nice if you test this method too 👍 to follow the project requirements 😊